### ISSUE TITLE
Weird rendering in nested directories

### ISSUE THREAD

**Issue by vyorkin** [isacikgoz/gitbatch#104]
First, I think this is a really great idea for a such tool. Thanks for working on it!

I'm using kitty terminal
```
font_family      JetBrains Mono
```

```sh
gitbatch --recursive-depth=3
```

![image](https://user-images.githubusercontent.com/988849/79616956-dbba8b80-80f5-11ea-8e6f-65d1b51e7535.png)

**Comment by sheb-gregor** [comment]
Same issue on Tilix Terminal:

![image](https://user-images.githubusercontent.com/11243996/105039638-75d6a800-5a69-11eb-9b7a-3e249d593e2c.png)

**Comment by isacikgoz** [comment]
This should be fixed by #113. Please feel free to re-open issue if you ever experience this again! And thanks a lot for the feedback!
